packages/record-tuple-playground/src/patch.js

```diff
@@ -14,6 +14,7 @@
  ** limitations under the License.
  */
 
+import "./wearkref-polyfill";
```

Comment:
Is the file name misspelled?

Reply:
File is misspelled, autocomplete avoided me an error...